docs/hub/spaces-sdks-streamlit.md

```diff
@@ -69,3 +69,13 @@ This Python script uses a [🤗 Transformers pipeline](https://huggingface.co/do
 <img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-hot-dog-streamlit.png"/>
 <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-hot-dog-streamlit-dark.png"/>
 </div>
+
+## Embed Steamlit Spaces on other webpages
+
+You can use the HTML `<iframe>` tag to embed a Streamlit Space as an inline frame on other webpages. Simply include the name of your Space in the URL, delimited with a dash `"-"` between the user/org and Space name, followed by the `.hf.space` suffix. For example, the demo above can be embedded in these docs with the following tag:
```

Comment:
this is not actually username dash space id (there's more characters to translate + collision handling)

```suggestion
You can use the HTML `<iframe>` tag to embed a Streamlit Space as an inline frame on other webpages. Simply include the URL of your Space, ending with the `.hf.space` suffix. To find the URL of your Space, you can use "Inspect element" in your browser, or use the "How to embed" button we'll ship shortly on all Spaces pages.

For example, the demo above can be embedded in these docs with the following tag:
```